packages/rx_bloc_cli/mason_templates/bricks/lib_permissions/__brick__/bin/server/controllers/permissions_controller.dart

```diff
@@ -21,7 +26,8 @@ class PermissionsController extends ApiController {
 
   Response permissionsHandler(Request request) { {{#has_authentication}}
     final headers = request.headers;
-    if (!headers.containsKey(AuthenticationService.authHeader)) {
+    // TODO: rewrite isTempUser check
+    if (!headers.containsKey(AuthenticationService.authHeader) || _usersService.isTempUser(_authenticationService.getUserIdFromAuthHeader(headers))) {
```

Comment:
Was this a forgotten comment or is this left for developers to implement on their own?

Reply:
It was a note to myself, since I had written the check differently initially in a local testapp. However I've decided against it, since this approach is better when considering the other feature flags we have. I've removed the comment